docs/developer-guide/nodejs-api.md

```diff
@@ -144,6 +144,15 @@ console.log(code.text);     // "var foo = bar;"
 
 In this way, you can retrieve the text and AST used for the last run of `linter.verify()`.
 
+## Linter
+
+This API can be used to have an immutable linter object to lint your code. `linter` API above is an instance of this class. For all the functionality, please refer to `linter` docs [above](#linter).
```

Comment:
We never discussed deprecation of `linter`, but i am open to that. But we need TSC for that and i think we can discuss that in a separate issue and deprecate it in next release.